packages/types/src/documents/plugin/index.ts

```diff
@@ -14,4 +16,17 @@ export interface FileType {
   name: string
 }
 
+export interface Plugin extends Document {
```

Comment:
Is this a global-db document? If so we should move the plugins folder into the global documents folder

Reply:
Good point, missed that!